index.js

```diff
@@ -260,6 +261,8 @@ exports.wants_skip = function (connection) {
 
     if (!plugin.cfg.check.authenticated && connection.notes.auth_user) return true;
 
+    if (connection.remote.is_local) return !plugin.cfg.check.local_ip;
```

Comment:
I'd prefer that this line of code not affect the flow except when its a positive hit. I find it much easier to reason about the flow this way:

```suggestion
    if (plugin.cfg.check.local_ip && connection.remote.is_local) return true;
```

Reply:
> this logic is hard to think through

Agreed! Does [this PR](https://github.com/analogic/haraka-plugin-rspamd/pull/1) help?  If not, feel free to take some liberties as I demonstrated in the PR. Perhaps a better name for the function. Or another type of flow control. Making the logic easier to reason about is welcome.